schema/productconfig.rnc

```diff
@@ -0,0 +1,475 @@
+# RELAX NG Schema for Docserv² product configuration
+
+# BASICS
+
+default namespace = ""
+
+# FIXME: Should this be <docservconfig> instead? (I guess it makes more sense
+# to be able to validate individual productconfig files instead of validating
+# an essentially internal representation)
+start = ds.productindex
+
+
+# TYPES
+
+ds.type.class =
```

Comment:
Is that pattern really needed?

Reply:
Counter-suggestion?

Reply:
Well, the [XHTML specification](https://www.w3.org/MarkUp/2010/xhtml-m12n-relaxng-20101216/#a_rmodule_XHTML_Datatypes) defines the `class` attribute as a simple RNG `string` datatype.

Reply:
For the sake of processing, I think I'll stick with the current format nevertheless.